tests/configuration/pages/pmmInventoryPage.js

```diff
@@ -334,4 +335,19 @@ module.exports = {
     I.click(this.fields.agentsLink);
     I.waitForVisible(agent, 30);
   },
+
+  async checkAgentsPresent(expectedAgentIds) {
+    const actualAgentIds = (await I.grabTextFromAll(this.fields.agentId))
+      .map((string) => string.replace('/agent_id/', ''));
+
+    if (expectedAgentIds.length !== actualAgentIds.length) {
```

Comment:
This could be just `assertNotEqual()` or if the aim is not to use "asserts" here than it should be `throw new Exception();`

Reply:
fixed